util/libcrypto.num

```diff
@@ -4448,3 +4448,10 @@ RSA_meth_get_multi_prime_keygen         4392	1_1_1	EXIST::FUNCTION:RSA
 RSA_meth_set_multi_prime_keygen         4393	1_1_1	EXIST::FUNCTION:RSA
 RAND_DRBG_get0_master                   4394	1_1_1	EXIST::FUNCTION:
 RAND_DRBG_set_reseed_time_interval      4395	1_1_1	EXIST::FUNCTION:
+ERR_load_SM2_strings                    5000	1_1_1	EXIST::FUNCTION:SM2
```

Comment:
I can see the reason to set a higher number, it makes it easier to fix conflicts as rebases happen...  We need to remember to reset this number before merging, though, as this is not exactly kosher (you've basically allocated 614 empty slots in a symbol vector on those platforms where the numbers are still meaningful).

Reply:
I think it was more likely a mistake, not realizing "make update" generates those numbers automatically.  This should be fixed.

Reply:
Indeed I did not realize `make update` handled this.

Reply:
Yeah well, I would say that it's a good idea to keep it this way as long as this PR is WIP, as any rebase with added stuff in this file won't force you to constantly renumber.  This, as long as we remember to scratch the change here before merging and just doing a new `make update` when merging...

(actually, this is probably a good idea generally, but it does mean that anyone merging needs to pay extra attention to the .num files)

Reply:
We haven't done this before.  Leaving big gaps is error prone.  Please stick with the right way to do it.  Until we discuss and change our minds on what is the right way :)

Reply:
@richsalz previously we used this gap to prevent `make update` causing conflicts on every master rebase, since the numbers are sequential. But indeed a big gap are error prone.

Reply:
Yeah, I get it.  But it starts at the highest number and doesn't look for gaps.  So things continue being a list with holes, which does impact some platforms as @levitte said.  What I do is rebase, edit util/libcrypto.num when there's a conflict to *remove the functions I added* and then do 'make update ; git add util/libcrypto.num ; git rebase --continue'

We should have a better solution.